.github/workflows/int-and-connected-test-run.yml

```diff
@@ -83,8 +83,7 @@ jobs:
           actions_subcommand: 'gradleinttest'
           pgport: ${{ job.services.postgres.ports[5432] }}
           test_to_run: 'check'
-          role_id: ${{ secrets.ROLE_ID }}
-          secret_id: ${{ secrets.SECRET_ID }}
+          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
```

Comment:
This is fine, although we could also read the secret in the called workflow. The only problem is if the secrets are stored differently in different repos that call the workflow, but I don't think that's the case here.

Reply:
Unfortunately, the called workflow doesn't have access to github secrets since it is a shell script.